spec/lib/jobs_spec.rb

```diff
@@ -2,12 +2,30 @@
 
 describe Jobs do
   describe '.search(options)' do
-    subject(:search) { Jobs.search(Keyword: 'jobs') }
+    subject(:search) do
+      Jobs.search({ query:'jobs',
+                    organization_codes: 'HE38',
+                    location_name: 'Washington, DC, USA',
+                    results_per_page: 10})
+    end
+
+    let(:usajobs_url) {'https://data.usajobs.gov/api/search'}
 
     it 'returns results' do
       expect(search.search_result.search_result_count).to eq 10
     end
 
+    it 'searches USAJOBS with the correct params' do
+      search
+      expect(a_request(:get, usajobs_url).with(
+        query: {
+          Keyword:        '',
+          Organization:   'HE38',
+          LocationName:   'Washington, DC, USA',
+          ResultsPerPage: 10}
```

Comment:
Fix curly bracket alignment, either:
```ruby
        query: {
          Keyword:        '',
          Organization:   'HE38',
          LocationName:   'Washington, DC, USA',
          ResultsPerPage: 10
        }
```

or:
```ruby
        query: { Keyword:        '',
                 Organization:   'HE38',
                 LocationName:   'Washington, DC, USA',
                 ResultsPerPage: 10 }
```

Reply:
bump

Reply:
this is out dated. Already fixed see how its flagged 'Outdated'

Reply:
I'm seeing this issue in the current version of this file:
https://github.com/GSA/search-gov/blob/b8c3c1f5e6d3539d19a12070b751b15c950762af/spec/lib/jobs_spec.rb#L19

Reply:
Look at it in the files changed tab.

Reply:
That's where I'm seeing it:
https://github.com/GSA/search-gov/pull/187/files#diff-42eb9f61e8fd848a9ea45dc8ae155816R23

You fixed the spacing issue, but not the bracket alignment.

Reply:
oh wait I think I see what you mean. I thought the issue was with the bracket.